baselines/fjord/fjord/conf/config.yaml

```diff
@@ -0,0 +1,8 @@
+---
+hydra:
+  run:
+    dir: ./runs/${now:%Y-%m-%d}:${now:%H-%M-%S}
+
+defaults:
+  - train_mode/fjord
+  - override hydra/job_logging: disabled
```

Comment:
```suggestion
loglevel: info
logfile: run.log

manual_seed: 123
model: resnet18
dataset: cifar10
num_clients: 100
data_path: "./data"
num_workers: 4
evaluate_every: 10

cuda: true
batch_size: 32
lr: 0.1
lr_scheduler: static
optimiser: sgd
momentum: 0
nesterov: false
weight_decay: 1e-4

sampled_clients: 10
min_fit_clients: 2
client_selection: random  # or balanced
num_rounds: 500
local_epochs: 1
strategy: fjord_fedavg
client_resources:
  num_cpus: 1
  num_gpus: 0.2
knowledge_distillation: ???
p_s:
  - 0.2
  - 0.4
  - 0.6
  - 0.8
  - 1.0
client_tier_allocation: uniform
```

Reply:
Unless you are thinking on adding more configs or variation between them, I think it's better to keep things simple. The above will run `fjord`, changing `knowledge_distillation` to true does `fjord_kd`.

Having it like this defined, simplifies usage i think. And for instance it allows your `run.sh` to be compactly executed as a single command:
```
bash
python -m fjord.main --multirun knowledge_distillation=true,false manual_seed=123,124,125
```
Which will create a dir structure like the below (with one sub dir per run):

```
(fjord-py3.10) ➜  fjord git:(fjord-flwr) ✗ ls multirun/2023-10-02/19-20-09
0  1  2  3  4  5  multirun.yaml
```

Reply:
Extensibility was the rationale behind this design.
I know that hydra can be intricate a times, but I think we don't hit this yet with this structure. 

Unless it's a hard requirement from your side, I would keep the structure as is. Let me know.